scalafix-core/src/main/scala/scalafix/internal/patch/ImportPatchOps.scala

```diff
@@ -200,7 +201,11 @@ object ImportPatchOps {
       allImports.filter(_.importers.forall(isRemovedImporter))
 
     def remove(toRemove: Tree): Patch = {
-      if (toRemove.pos == Position.None) return Patch.empty
```

Comment:
This brings false negatives now that quasiquotes have positions (since https://github.com/scalameta/scalameta/issues/3425). This is effectively a very minor breaking change in the API.